bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/Ballot/ICiphertextContest.cs

```diff
@@ -1,23 +1,13 @@
-namespace ElectionGuard.Encryption.Ballot
-{
-    /// <summary>
-    /// A contest on a ballot or in a tally
-    /// </summary>
-    public interface IElectionContest
-    {
-        /// <summary>
-        /// The object id
-        /// </summary>
-        string ObjectId { get; }
+using System.Collections.Generic;
 
-        /// <summary>
-        /// The sequence order
-        /// </summary>
-        ulong SequenceOrder { get; }
+namespace ElectionGuard.Ballot
+{
 
+    public interface ICiphertextContest : IElectionContest
```

Comment:
interface segregation has been a big help downstream in decryption to minimize the amount of dependencies on things like .UI.Lib and having to copy objects around